omniscidb/ResultSet/ResultSet.cpp

```diff
@@ -256,10 +256,13 @@ std::string ResultSet::getStrScalarVal(const ScalarTargetValue& current_scalar,
         if (col_type->isDate()) {
           oss << getStrDateFromSeconds(boost::get<int64_t>(current_scalar));
         } else if (col_type->isTime()) {
-          oss << getStrTimeFromSeconds(boost::get<int64_t>(current_scalar));
+          oss << getStrTStamp(boost::get<int64_t>(current_scalar),
+                              col_type->as<hdk::ir::TimeType>()->unit())
+              << " scalar: " << current_scalar;
```

Comment:
I think this function is used to print result sets in PyHDK, scalar values would be really unexpected there.

Reply:
`getStrTStamp` print all time types as timestamp, it looks like a debug function. If it's for end user we should rewrite it.